docs/feature-gates.md

```diff
@@ -446,6 +446,8 @@ Refer to this [document](egress.md#subnetinfo) for more information.
 `L7FlowExporter` enables users to export application-layer flow data using Pod or Namespace annotations.
 Refer to this [document](network-flow-visibility.md#l7-visibility) for more information.
 
+Note: `L7FlowExporter` is deprecated in Antrea v2.5.0, and will be removed in v2.6.0.
```

Comment:
I think that for visibility, we should set the `Stage` to `Deprecated` in the main table at the top of the document

Reply:
Updated